src/utils/session-id.js

```diff
@@ -16,7 +17,11 @@ const generateSessionId = () => uuidv4()
  */
 const saveSessionIdInCookie = (cookieName, sessionId, hasExpirationDate) => {
   if (hasExpirationDate) {
-    Cookie.set(cookieName, sessionId, { expires: COOKIE_EXPIRY_DAYS })
+    Cookie.set(COOKIE_PREFIX + cookieName, sessionId, {
```

Comment:
Just a personal preference if I may point out one:
```suggestion
    Cookie.set(`${COOKIE_PREFIX}${cookieName}`, sessionId, {
```